django_lti_tool_provider/__init__.py

```diff
@@ -7,11 +7,14 @@
 logging.getLogger("").addHandler(logging.NullHandler())
 
 
+# pylint: disable=unused-argument
+# This class specifies method signatures; while pylint is intelligent enough to ignore unused args in abstractmethods
+# methods with default implementations are still reported. Hence we suppress unused-argument for entire class
 @add_metaclass(ABCMeta)
 class AbstractApplicationHookManager(object):
     """ Class that performs authentication and redirection for LTI views """
     @abstractmethod
-    def authentication_hook(self, request, user_id=None, username=None, email=None):
+    def authentication_hook(self, request, user_id=None, username=None, email=None, **kwargs):
```

Comment:
@jbzdak please elaborate further, especially how it is differ from when no additional keyword arguments are passed (`kwargs` should be just empty than)